.travis.yml

```diff
@@ -1,44 +1,50 @@
 sudo: required
+language: generic
 
 branches:
   only:
     - master
 
+# Cache is irrelevant for linux/docker builds, but useful to speed up osx
 cache:
   directories:
     - $HOME/.pyenv
-    - googletest
-    - swig-3.0.10
+    - $HOME/googletest
 
 matrix:
   include:
   - os: linux
-    language: generic
-    env: TOXENV=py34
+    services: docker
+    env:
+      - DOCKER_IMAGE=vinther/tick_manylinux1_x86_64:1.2
```

Comment:
No, tick_manylinux1_x86_64 and tick_ubuntu are just images where tick can be built without the need to install dependencies. The dependencies are already installed in the images.